spec/index.html

```diff
@@ -7406,107 +7406,112 @@ <h5>TZ</h5>
         </section>
 
         <!-- QT -->
-        <section id="func-quoted-triples">
-          <h4>Functions on Quoted Triples</h4>
+        <section id="func-reified-triples">
+          <h4>Functions on Reified Triples</h4>
```

Comment:
@rubensworks  The change that your recent commit https://github.com/w3c/sparql-query/pull/149/commits/118528fb28ee78efe419facac3e903431f0f5b9a implements here is wrong. The functions in this section are not for "reified triples" but for triple terms.
```suggestion
        <section id="func-triple-terms">
          <h4>Functions on Triple Terms</h4>
```

Reply:
After @gkellogg's comment above, my reasoning for this change was that readers might be more familiar with the "reified triples" terminology, and would easier find their way to this part in the spec. But you're right that it's not accurate, so let's revert.